test/e2e/tests/onboarding/onboarding.spec.js

```diff
@@ -270,76 +271,122 @@ describe('MetaMask onboarding @no-mmi', function () {
-          chainId.toString(),
-        );
-        await driver.fill(
-          '[data-testid="network-form-ticker-input"]',
-          currencySymbol,
-        );
+          await driver.clickElement({
+            text: 'General',
+          });
+          await driver.delay(largeDelayMs);
+          await driver.clickElement({ text: 'Add a network' });
 
-        // Add rpc url
-        const rpcUrlInputDropDown = await driver.waitForSelector(
-          '[data-testid="test-add-rpc-drop-down"]',
-        );
-        await rpcUrlInputDropDown.click();
-        await driver.delay(tinyDelayMs);
-        await driver.clickElement({
-          text: 'Add RPC URL',
-          tag: 'button',
-        });
-        const rpcUrlInput = await driver.waitForSelector(
-          '[data-testid="rpc-url-input-test"]',
-        );
-        await rpcUrlInput.clear();
-        await rpcUrlInput.sendKeys(networkUrl);
-        await driver.clickElement({
-          text: 'Add URL',
-          tag: 'button',
-        });
+          await driver.waitForSelector(
+            '.multichain-network-list-menu-content-wrapper__dialog',
+          );
 
-        await driver.clickElement({ text: 'Save', tag: 'button' });
-        await driver.clickElement({
-          text: 'Done',
-          tag: 'button',
-        });
+          await driver.fill(
+            '[data-testid="network-form-network-name"]',
+            networkName,
+          );
+          await driver.fill(
+            '[data-testid="network-form-chain-id"]',
+            chainId.toString(),
+          );
+          await driver.fill(
+            '[data-testid="network-form-ticker-input"]',
+            currencySymbol,
+          );
 
-        await driver.clickElement('.mm-picker-network');
-        await driver.clickElement(
-          `[data-rbd-draggable-id="${toHex(chainId)}"]`,
-        );
+          // Add rpc url
+          const rpcUrlInputDropDown = await driver.waitForSelector(
+            '[data-testid="test-add-rpc-drop-down"]',
+          );
+          await driver.delay(tinyDelayMs);
+          await rpcUrlInputDropDown.click();
+          await driver.delay(tinyDelayMs);
+          await driver.clickElement({
+            text: 'Add RPC URL',
+            tag: 'button',
+          });
+          const rpcUrlInput = await driver.waitForSelector(
+            '[data-testid="rpc-url-input-test"]',
+          );
+          await rpcUrlInput.clear();
+          await rpcUrlInput.sendKeys(networkUrl);
+          await driver.clickElement({
+            text: 'Add URL',
+            tag: 'button',
+          });
+
+          await driver.clickElement({ text: 'Save', tag: 'button' });
+
+          await driver.delay(largeDelayMs);
+          await driver.waitForSelector('[data-testid="category-back-button"]');
+          const generalBackButton = await driver.waitForSelector(
+            '[data-testid="category-back-button"]',
+          );
+          await generalBackButton.click();
```

Comment:
```suggestion
          await driver.clickElement('[data-testid="category-back-button"]');
```
same reason as above